README.md

```diff
@@ -1,10 +1,75 @@
 # indexer
 
-Blockchain indexer and database
+This project indexes data from the regen ledger and stores it in a postgresql database.
+It is built as single python script that spawns independent processes for indexing application specific data.
+This database is currently being used for several purposes in relation to the Regen Marketplace and the Regen Groups UI.
+
+## GraphQL API
+
+The regen-server provides a GraphQL API for accessing the indexer database:
+
+https://api.regen.network/indexer/v1/graphiql
+
+Both the groups-ui and the regen-marketplace use this API to access the data.
+
+## Dependencies
+
+In order to run this project locally you will need the following:
+
+1. a local testnet version of regen-ledger with REST API and RPC enabled
```

Comment:
should we add a link to https://docs.regen.network/ledger/get-started/local-testnet.html?